crates/bitwarden-wasm/src/client.rs

```diff
@@ -37,10 +37,10 @@ impl BitwardenClient {
     #[wasm_bindgen(constructor)]
     pub fn new(settings_input: Option<String>, log_level: Option<LogLevel>) -> Self {
         console_error_panic_hook::set_once();
-        if let Err(e) =
+        if let Err(_e) =
             console_log::init_with_level(convert_level(log_level.unwrap_or(LogLevel::Info)))
         {
-            panic!("failed to initialize logger: {:?}", e);
+            warn!("Logger already initialized, log level will not be changed.");
```

Comment:
It's probably going to be confusing to get a warning when you setup multiple sdk clients. If we do want to warn about this we should check if the log level is different.

Reply:
Agreed, at the time of looking at the `console_log` crate, I didn't think it was possible. 

After further research, I found out we can use the log crate functions, and we can just set the level. 

Thoughts on this approach?

From an SDK consumer perspective, this seems ideal.

https://docs.rs/log/0.4.22/log/fn.set_max_level.html

Edit here https://github.com/bitwarden/sdk/pull/935/commits/3a86e8e51083021fd593b0d28ba34dd5d819fed7